### DISCUSSION TITLE
Escalating events via SOC doesn't show any observables in the case

### DISCUSSION THREAD

**Discussion by pd98** [Security-Onion-Solutions/securityonion#5747] [Unsupported Versions] [answered]
Hi,

I escalated a log entry to the hive via hunt, but I could not find any observables like source IP or destination IP mentioned in the created case. 
Is there any way to automatically add certain observables when the case is created via SOC?

Details :

1. Security Onion Version : 2.3.61
2. Install source : ISO
3. Install type : standalone
4. so-status : all ok

Any help would be appreciated. Thanks

**Comment by clairmont32** [comment]
I'm seeing this with 2.3.80 as well. Escalating a Suricata alert simply puts the json logs into the description of the case with no other Hive fields set.

**Comment by jertel** [comment, marked as answer]
I escalated an alert just now, on 2.3.80, and the case was successfully created in TheHive. As exepected, it has the following:

- a title correctly reflecting the name of the rule
- a tag correctly showing "SecurityOnion"
- Amber TLP
- Description correctly set to the alert group summary fields, including 'count', 'severity', and 'module'.

This is the same behavior as previously releases. It's not perfect but, as we mentioned at the Security Onion conference on Friday, case management is an area we're looking at improving over the next year.

If you're experiencing a different behavior from what I described above please provide more information, including screenshots of the alerts page prior to clicking the Escalate button, and then the resulting TheHive screen showing the case missing fields (and note the specific fields).

If you'd like to see more details about the underlying event in the escalated case's description JSON value then I suggest drilling down into the alert group and Escalating one of the group's events, and then Acknowledge the group to remove the remaining events.

**Reply by clairmont32** [reply]
Ok so no observables are being set then. We'll have to ensure we're parsing the json from the description then with some other automation.

**Comment by clairmont32** [comment]
Unfortunately, I'm not entirely convinced this is actually answered after reviewing the SOAR video from the conference. @weslambert mentions performing actions based on observables from the hive within a SOAR platform. Currently, from my testing on .80, there are no observables ever filed in a case and only the description contains information in json blob.

Reference time 17:25 from the presentation.
https://youtu.be/zf9jw9x0IUo?t=1045
